apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -732,7 +739,7 @@ export class ExtractorConfig {
           // Use the manually specified "<lookup>" value
           projectFolder = options.projectFolderLookupToken;
 
-          if (!FileSystem.exists(options.projectFolderLookupToken)) {
+          if (!options.ignoreMissingConfigTargets && !FileSystem.exists(options.projectFolderLookupToken)) {
```

Comment:
It'd be weird if this one didn't exist.

Seems like this option really should only apply to other tools' outputs and not any file. The option name should be updated to reflect that too.